rules.js

```diff
@@ -186,6 +186,103 @@ class sub_with_return extends rule(3) {
     }
 }
 
+class object_formatting extends rule(4) {
+    constructor() {
+        super()
+        this._node = 'object'
+        this.message = ``
+        this.spaces_after_colon = 1
+        this.spaces_before_colon = 0
+        this.trailing_commas = 'no' // 'no', 'required'
+        this.maxOneLine = 4
+        this.key_in_quotes = 'consistent' // 'allways', 'consistent', 'any'
```

Comment:
I'm a little unclear on how this is supposed to work, `key_in_quotes` and `trailing_commas` are not set anywhere else. Is the intent here that you're supposed to replace them with the other predefined values depending on your preferences? Also, minor nitpick but "allways" is misspelled.

Reply:
Those values can be changed in `brslint.config`